.yarnrc.yml

```diff
@@ -11,7 +15,5 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-allow-scripts.cjs
     spec: "https://raw.githubusercontent.com/LavaMoat/LavaMoat/main/packages/yarn-plugin-allow-scripts/bundles/@yarnpkg/plugin-allow-scripts.js"
-  - path: .yarn/plugins/@yarnpkg/plugin-constraints.cjs
-    spec: "@yarnpkg/plugin-constraints"
 
-yarnPath: .yarn/releases/yarn-3.2.1.cjs
+yarnPath: .yarn/releases/yarn-4.1.1.cjs
```

Comment:
Heads up: We don't need to include the Yarn binary in the repo anymore with Yarn 4; [it defaults to using Corepack now](https://yarnpkg.com/blog/release/4.0). We can make that change in another PR, but I thought I'd point that out.